Payjoin-spec.md

```diff
@@ -70,6 +70,42 @@ There is no guarantee concerning the ordering of outputs and inputs in either th
 Note: We strongly recommend the sender to submit a PSBT in Base64 format.
 However, we also accept the PSBT or a bitcoin transaction in hex format.
 
+### Optional parameters
+
+When the payjoin sender sends the POST request to the payjoin endpoint, he can optionally specifies the following parameters:
+
+* `v=`, the version number of the payjoin protocol that the client is using. The current version is `1`.
```

Comment:
Sorry for being the dumb one around here, but I don't actually know concretely how this is implemented (probably just lack of experience with HTTP?). Currently I send a base64 PSBT in the body of the request, how do I add optional parameters like `v=`? Also in the body, I take it, but what format?

Reply:
query parameter:

`https://btcpay.example.com/payjoin`

Would become

`https://btcpay.example.com/payjoin?v=1&noadjustfee=true`

Reply:
Thanks, makes sense.